libs/devices/harmony/routes/src/lib/screens/manage-files/harmony-manage-files.types.tsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import { FileManagerFile } from "devices/common/ui"
+
+export enum FileCategoryId {
+  alarmFiles = "alarmFiles",
+  relaxationFiles = "relaxationFiles",
```

Comment:
Nitpick, but the convention in the project is to start enum member names with an uppercase letter.

```suggestion
  AlarmFiles = "alarmFiles",
  RelaxationFiles = "relaxationFiles",
```

Reply:
8ce35d97